wa/workloads/gfxbench/__init__.py

```diff
@@ -20,8 +20,9 @@
 
 class Gfxbench(ApkUiautoWorkload):
 
-    name = 'gfxbench-corporate'
-    package_names = ['net.kishonti.gfxbench.gl.v50000.corporate']
+    name = 'gfxbench'
+    package_names = ['com.glbenchmark.glbenchmark27']
```

Comment:
Sorry if this is an obvious question but is this the correct package name for the non-corporate gfxbenchmark application? If so how does it differ to our `glbenchmark` workload?

https://github.com/ARM-software/workload-automation/blob/72298ff9acb4eecca061261ee09567926f4df8db/wa/workloads/glbenchmark/__init__.py#L58

Reply:
Hmm, good question. That's odd. I tried this on Pixel 6 and Oneplus Nord and it worked fine. glbenchmark refused to start on Pixel 6 due to version mismatch, haven't tried on phone.

It seems gfxbenchmark supersedes what used to be known as glbenchmark

https://kishonti.net/graphics-benchmarking.jsp

we can put explicit version check to help. I am not sure what would be the best way to handle this tbh

Reply:
I must clarify. I didn't install glbench, just tried to run this workload with gfxbench installed to see if it'll start it instead. It didn't and got a complain of wrong version instead.

Reply:
Right I see, so you're using the non corporate version of gfxbench mark in your testing but that didn't work correctly with either the `gfxbench` or `glbench` workload? Do you have the error output generated / what version it is reporting? 
The two apps I've seen for these workloads have greatly differing UI's so I'm just trying to figure out which version you require.

Reply:
> Right I see, so you're using the non corporate version of gfxbench mark in your testing but that didn't work correctly with either the gfxbench or glbench workload?

Reply:
Right ok, this makes sense. Agreed, if we have a similar version check for 4.X/5.X as we do for the glbenchmark workload has for v2.7/2.5 this would prevent any overlap between the two workloads and provide the appropriate error message if the wrong application is available on the device.

Reply:
Okay. Looking at other examples, it seems adding `supported_versions = ['4', '5']` is enough. Tested that with only '4' in the supported version and I got a complaint about v 4 not preset. Once I added '5', it worked. I fixed the 2 required 2 lines complaint flagged by the automatic checks and pushed the updated branch.